tester/main.cpp

```diff
@@ -301,47 +300,41 @@ int main(int argc, char const *argv[]) {
 	// preparing versions
 	vc::Version::Builder builder;
 	std::list<run_element_t> jobList;
+	vc::vc_utils_init();
 
 	for (const auto &size : size_list) {
 		for (const auto &cat : configList) {
 			for (const auto &k : cat.kernel_config_list) {
 				for (const auto &datatype : k.data_type_option_list) {
 					const std::string label = cat.category_label + " " +
-					                          k.kernel_label + " - " +
-					                          size.first + " - " +
-					                          datatype.first;
-					builder._compiler = default_compiler;
+						k.kernel_label + " - " +
+						size.first + " - " +
+						datatype.first;
 					const std::string kernelSourceDir = polybench_source + "/" +
-					                                    cat.category_folder + "/" +
-					                                    k.kernel_folder;
-					vc::Option sourceDir("sourceDir", "-I", kernelSourceDir);
-					builder._optionList = {size.second, datatype.second, sourceDir};
-					builder._optionList.insert( builder._optionList.end(),
-					                            default_options.begin(),
-					                            default_options.end());
-					builder._fileName_src = kernelSourceDir + "/" + k.fileName;
-					builder._functionName = k.function_name;
-					jobList.push_back(std::make_pair(label, builder.build()));
+						cat.category_folder + "/" +
+						k.kernel_folder;
+					const vc::opt_list_t options = {
+						vc::make_option("-include polybench.c"),
```

Comment:
the include of the C file should no longer be required.
Instead, there should be the possibility to list more than one file as version source file.
To this end, libVC low level APIs are required (use the builder instead of `vc::createVersion`)